logux-vue-todomvc/.browserslistrc

```diff
@@ -0,0 +1,3 @@
+> 1%
```

Comment:
I recommend using the default Browserslist config. If you Vue CLI force us to create Browserslist config, put `defaults` there.

Reply:
It's generated by Vue CLI, too.

Reply:
Let’s fix it since Browserslist is part of our family :).